### PR TITLE
Add new test contact us from the order confirmation page

### PR DESCRIPTION
**Pull request by SD1982**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x 
| Description?      | Add a new test "contact us from the order confirmation page"
| Type?             | refacto 
| Category?         | TE 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | nothing
| How to test?      | test with this command `URL_FO='your FO url' DB_SERVER="your DB server" TEST_PATH="functional/FO/12_orderConfirmation/05_contactUs" npm run specific-test`
| Possible impacts? | any

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24160)

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/12_orderConfirmation/05_contactUs.js [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+const {DefaultCustomer} = require('@data/demo/customer');
+const CartData = require('@data/FO/cart');
+const {PaymentMethods} = require('@data/demo/paymentMethods');
+
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+// context
+const baseContext = 'functional_FO_contact_us_from_order_confirmation_page';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/12_orderConfirmation/05_contactUs.js]:
```suggestion
const baseContext = 'functional_FO_orderConfirmation_contactUs';
```

File: themes/classic/templates/checkout/order-confirmation.tpl [hunk 2]
```diff
@@ -57,7 +57,9 @@
           <div id="order-details" class="col-md-4">
             <h3 class="h3 card-title">{l s='Order details' d='Shop.Theme.Checkout'}:</h3>
             <ul>
-              <li>{l s='Order reference: %reference%' d='Shop.Theme.Checkout' sprintf=['%reference%' => $order.details.reference]}</li>
+              <li>Order reference:
+                <span id="order-reference-value">{l s='%reference%' d='Shop.Theme.Checkout' sprintf=['%reference%' => $order.details.reference]}</span>
+              </li>
```
Comment (Progi1984) [inline themes/classic/templates/checkout/order-confirmation.tpl]: Please don't do that : `Order reference:` is no more translated

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: themes/classic/templates/checkout/order-confirmation.tpl (hunk 2 above)
Comment (boubkerbribri) [reply themes/classic/templates/checkout/order-confirmation.tpl]: Is there a solution to separate the label from the value? or should we use regex to do it?

Comment (Progi1984) [reply themes/classic/templates/checkout/order-confirmation.tpl]: Ping @Julievrz

Comment (Progi1984) [reply themes/classic/templates/checkout/order-confirmation.tpl]: We need to remove the reference from the wording.

Comment (Julievrz) [reply themes/classic/templates/checkout/order-confirmation.tpl]: You mean creating a string "Order reference:" in Shop.Theme.Checkout and get rid of the "%reference%" in the existing string "Order reference: %reference%"?

Comment (SD1982) [reply themes/classic/templates/checkout/order-confirmation.tpl]: I have a new solution the .tpl is now as before :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]
LGTM

**Comment by Progi1984** [comment]
Thanks @SD1982

**Pull request merged**